client/blocks/inline-help/inline-help-more-resources.tsx

```diff
@@ -1,78 +1,133 @@
 			<ul className="inline-help__more-resources" aria-labelledby="inline-help__more-resources">
-				<ResourceItem
-					link={ localizeUrl( 'https://wordpress.com/support/video-tutorials/' ) }
-					icon="play"
-					text={ __( 'Video tutorials' ) }
-					svgColor="#C9356E"
-				/>
-				<ResourceItem
-					link={ localizeUrl( 'https://wordpress.com/webinars' ) }
-					icon="video-camera"
-					text={ __( 'Webinars' ) }
-					onClickHandler={ trackCoursesButtonClick }
-					svgColor="#E68B28"
-				/>
-				<ResourceItem
-					link={ 'https://wpcourses.com/?ref=wpcom-help-more-resources' }
-					icon="computer"
-					text={ __( 'Courses' ) }
-					svgColor="#09B585"
-				/>
-				<ResourceItem
-					link={ 'https://learn.wordpress.com' }
-					icon="list-ordered"
-					text={ __( 'Step-by-step guides' ) }
-					svgColor="#B35EB1"
-				/>
+				<li className="inline-help__resource-item">
+					<div className="inline-help__resource-cell">
+						<a
+							href={ localizeUrl( 'https://wordpress.com/support/video-tutorials/' ) }
+							rel="noreferrer"
+							target="_blank"
+							className="inline-help__video"
+						>
+							<Icon icon={ video } size={ 24 } />
+							<span>{ __( 'Video tutorials' ) }</span>
+						</a>
+					</div>
+				</li>
+				<li className="inline-help__resource-item">
+					<div className="inline-help__resource-cell">
+						<a
+							href={ localizeUrl( 'https://wordpress.com/webinars' ) }
+							rel="noreferrer"
+							target="_blank"
+							onClick={ trackCoursesButtonClick }
+							className="inline-help__capture-video"
+						>
+							<Icon icon={ captureVideo } size={ 24 } />
+							<span>{ __( 'Webinars' ) }</span>
+						</a>
+					</div>
+				</li>
+				<li className="inline-help__resource-item">
+					<div className="inline-help__resource-cell">
+						<a
+							href={ localizeUrl( 'https://wpcourses.com/?ref=wpcom-help-more-resources' ) }
+							rel="noreferrer"
+							target="_blank"
+							className="inline-help__desktop"
+						>
+							<Icon icon={ desktop } size={ 24 } />
+							<span>{ __( 'Courses' ) }</span>
+						</a>
+					</div>
+				</li>
+				<li className="inline-help__resource-item">
+					<div className="inline-help__resource-cell">
+						<a
+							href={ localizeUrl( 'https://learn.wordpress.com' ) }
+							rel="noreferrer"
+							target="_blank"
+							className="inline-help__format-list-numbered"
+						>
+							<Icon icon={ formatListNumbered } size={ 24 } />
+							<span>{ __( 'Step-by-step guides' ) }</span>
+						</a>
+					</div>
+				</li>
+				<li className="inline-help__resource-item">
+					<div className="inline-help__resource-cell">
+						<Button
+							isLink
+							onClick={ () => handleWhatsNewClick() }
+							className="inline-help__new-releases"
+						>
+							<MaterialIcon icon="new_releases" size={ 24 } />
+							<span>{ __( "What's new" ) }</span>
+							{ showWhatsNewDot && <Icon icon={ circle } size={ 16 } fill="#0675C4" /> }
```

Comment:
Since this is is in Calypso, maybe use `var( --color-masterbar-unread-dot-background )`? just like the notifications bubble.